transport_udp.go

```diff
@@ -44,9 +44,19 @@ type udpSender struct {
 	thriftProtocol  thrift.TProtocol
 	process         *j.Process
 	processByteSize int
+
+	// reporterStats provides access to stats that are only known to Reporter
+	reporterStats reporterstats.ReporterStats
+
+	// The following counters are always non-negative, but we need to send them in signed i64 Thrift fields,
+	// so we keep them as signed. At 10k QPS, overflow happens in about 300 million years.
```

Comment:
nice, let's hope Jaeger will still be around after 300 million years! :)

Reply:
and running without a restart